includes/Core/Assets/Assets.php

```diff
@@ -668,20 +669,22 @@ private function get_inline_base_data() {
 		global $wpdb;
 		$site_url     = $this->context->get_reference_site_url();
 		$current_user = wp_get_current_user();
+		$modules      = new Modules( $this->context );
 
 		$inline_data = array(
-			'homeURL'          => trailingslashit( $this->context->get_canonical_home_url() ),
-			'referenceSiteURL' => esc_url_raw( trailingslashit( $site_url ) ),
-			'userIDHash'       => md5( $site_url . $current_user->ID ),
-			'adminURL'         => esc_url_raw( trailingslashit( admin_url() ) ),
-			'assetsURL'        => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
-			'blogPrefix'       => $wpdb->get_blog_prefix(),
-			'ampMode'          => $this->context->get_amp_mode(),
-			'isNetworkMode'    => $this->context->is_network_mode(),
-			'timezone'         => get_option( 'timezone_string' ),
-			'siteName'         => get_bloginfo( 'name' ),
-			'enabledFeatures'  => Feature_Flags::get_enabled_features(),
-			'webStoriesActive' => defined( 'WEBSTORIES_VERSION' ),
+			'homeURL'              => trailingslashit( $this->context->get_canonical_home_url() ),
+			'referenceSiteURL'     => esc_url_raw( trailingslashit( $site_url ) ),
+			'userIDHash'           => md5( $site_url . $current_user->ID ),
+			'adminURL'             => esc_url_raw( trailingslashit( admin_url() ) ),
+			'assetsURL'            => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
+			'blogPrefix'           => $wpdb->get_blog_prefix(),
+			'ampMode'              => $this->context->get_amp_mode(),
+			'isNetworkMode'        => $this->context->is_network_mode(),
+			'timezone'             => get_option( 'timezone_string' ),
+			'siteName'             => get_bloginfo( 'name' ),
+			'enabledFeatures'      => Feature_Flags::get_enabled_features(),
+			'webStoriesActive'     => defined( 'WEBSTORIES_VERSION' ),
+			'psiManuallyActivated' => $modules->manually_enabled( 'pagespeed-insights' ),
```

Comment:
Instead of adding this information here, let's better update the appropriate hook in the Modules class: https://github.com/google/site-kit-wp/blob/740fa3262c465dafd097a7b8d4100bf68c6a52c4/includes/Core/Modules/Modules.php#L247-L262

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/a76c3de66869fc93743b5ac7b48a1bdf044d7d26